tests/rptest/services/openmessaging_benchmark_configs.py

```diff
@@ -110,84 +110,108 @@ def validate_metrics(metrics, validator):
         "name": "simple-driver",
         "replication_factor": 3,
         "request_timeout": 300000,
-        "enable_idempotence": "false",
-        "acks": "1",
-        "linger_ms": 1,
-        "max_in_flight": 1,
-        "batch_size": 131072,
-        "auto_offset_earliest": "earliest",
-        "auto_commit": "false",
-        "max_partition_fetch": 131072
+        "producer_config": {
+            "enable.idempotence": "false",
+            "acks": "1",
```

Comment:
Good point, all the tests in this PR will pass with the `ACK_ALL_GROUP_LINGER_1MS_IDEM_MAX_IN_FLIGHT` driver so I'll probably switch them to that by default. It's probably better from a testing prospective as well since it stresses more code paths.

Reply:
Swapped to using `ACK_ALL_GROUP_LINGER_1MS_IDEM_MAX_IN_FLIGHT` by default.

Reply:
If we aren't using it, can we delete it? The name doens't really mean anything, it's acks=1, ...

Reply:
From a quick grep it looks like it is in use;
```
tests/rptest/scale_tests/many_partitions_test.py:869:            "SIMPLE_DRIVER",
tests/rptest/services/openmessaging_benchmark_configs.py:109:    SIMPLE_DRIVER = {
tests/rptest/services/openmessaging_benchmark_configs.py:284:        "SIMPLE_DRIVER":
tests/rptest/services/openmessaging_benchmark_configs.py:285:        SIMPLE_DRIVER,
grep: tests/rptest/services/__pycache__/openmessaging_benchmark.cpython-310.pyc: binary file matches
grep: tests/rptest/services/__pycache__/openmessaging_benchmark_configs.cpython-310.pyc: binary file matches
tests/rptest/services/openmessaging_benchmark.py:159:                 driver: str | dict[str, Any] = "SIMPLE_DRIVER",
grep: tests/rptest/tests/__pycache__/services_self_test.cpython-310.pyc: binary file matches
tests/rptest/tests/services_self_test.py:42:    @matrix(driver=["SIMPLE_DRIVER"], workload=["SIMPLE_WORKLOAD"])
tests/rptest/redpanda_cloud_tests/high_throughput_test.py:1009:            self._ctx, self.redpanda, "SIMPLE_DRIVER",
```
Definitely doesn't seem like we should be using it though. Will open up a separate PR to remove it and it's uses.